.github/workflows/pull_request.yml

```
@@ -0,0 +1,30 @@
+name: Pull Request
+
+on:
+  pull_request:
+    branches:
+      - master
```

This on section should look like the following. Otherwise the workflow starts with anything that you change on a PR (change reviewers, add comment, ...). Also I added `push` events in case someone pushes directly on master.
```suggestion
on:
  push:
    branches: master
  pull_request:
    types: [opened, reopened, ready_for_review, synchronize]
    branches: master
```

---

Ah Ok, I thought the types were the "labels", that's I ignored it.